xml/admin_operating_pools.xml

```diff
@@ -64,825 +65,827 @@
-    <varlistentry>
-     <term>crush_ruleset_name</term>
-     <listitem>
-      <para>
-       The name of the crush ruleset for this pool. If the specified ruleset
-       does not exist, the creation of replicated pools will fail with -ENOENT.
-       For replicated pools it is the ruleset specified by the <varname>osd
-       pool default crush replicated ruleset</varname> configuration variable.
-       This ruleset must exist. For erasure pools it is 'erasure-code' if the
-       default erasure code profile is used or
-       <replaceable>POOL_NAME</replaceable> otherwise. This ruleset will be
-       created implicitly if it does not exist already.
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>erasure_code_profile=profile</term>
-     <listitem>
-      <para>
-       For erasure coded pools only. Use the erasure code profile. It must be
-       an existing profile as defined by <command>osd erasure-code-profile
-       set</command>.
-      </para>
-      <para>
-       When you create a pool, set the number of placement groups to a
-       reasonable value. Consider the total number of placement groups per OSD
-       too. Placement groups are computationally expensive, so performance will
-       degrade when you have many pools with many placement groups (for example
-       50 pools with 100 placement groups each).
-      </para>
-      <para>
-       See <xref linkend="op-pgs"/> for details on calculating an appropriate
-       number of placement groups for your pool.
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>expected_num_objects</term>
-     <listitem>
+  <variablelist>
+   <varlistentry>
+    <term>POOL_NAME</term>
+    <listitem>
+     <para>
+      The name of the pool. It must be unique. This option is required.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>PG_NUM</term>
+    <listitem>
+     <para>
+      The total number of placement groups for the pool. This option is
+      required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>PGP_NUM</term>
+    <listitem>
       <para>
-       The expected number of objects for this pool. By setting this value
-       (together with a negative <option>filestore merge threshold</option>),
-       the PG folder splitting happens at the pool creation time. This avoids
-       the latency impact with a runtime folder splitting.
+        In previous versions of &productname;, it was necessary to set the number
+        of placement groups on each pool explicitly. In &productname; &productnumber;,
+        the autoscaler is active by default, so the number of placement groups is
+        set and adjusted automatically. For more information on the autoscaler,
+        see <xref linkend="op-pgs-autoscaler"/>.
       </para>
-     </listitem>
-    </varlistentry>
-   </variablelist>
-  </sect2>
+    </listitem>
```

Comment:
Add the pool type argument here? This would be the place for it in the full argument list.